RecoTracker/TrackProducer/interface/DAFTrackProducerAlgorithm.h

```diff
@@ -8,6 +8,10 @@
 #ifndef DAFTrackProducerAlgorithm_h
 #define DAFTrackProducerAlgorithm_h
 
+
+#include "AlgoProductTraits.h"
```

Comment:
subsystem/package path may be more appropriate for files in the /interface subdir